.ci/azure-pipelines.yml

```diff
@@ -0,0 +1,78 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+  tags:
+    include:
+      - "*"
+pr:
+  branches:
+    include:
+      - "*"
+
+jobs:
+  - job: Build
+    displayName: "Build"
+
+    pool:
+      vmImage: "ubuntu-latest"
+
+    steps:
+      - task: NodeTool@0
+        displayName: "Install node"
+        inputs:
+          versionSpec: "12.x"
```

Comment:
We are using single quotes, title casing for the display name, and no indentation for arrays in these YAML files to keep a consistent style regardless of language.

Reply:
Well this project is using double quotes all over the place so I thought it might be good to keep it consistent

Reply:
I'll add json and yaml files to prettier soon anyway then it'll handle the formatting

Reply:
Changed the display name, the formatting using prettier will be fixed in a followup PR that is already prepared